src/Components/App/App.js

```diff
@@ -31,28 +34,43 @@ class App extends React.Component {
   fetchMovies = () => {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
     .then(response => {
-      console.log(response)
       if (!response.ok) {
         throw Error('Restuffing Burrito... sorry about that.')
       } else {
         return response.json()
       }
     })
     .then(data => {
-      console.log(data)
-      this.setState({ movies: data.movies, error: false, errorMessage: ''  })
+      this.setState({ movies: data.movies, error: false, errorMessage: '', currentMovie: null })
     })
     .catch(error => {
-      this.setState({ movies: [], error: true, errorMessage: error.message })
+      this.setState({ movies: [], error: true, errorMessage: error.message, currentMovie: null })
+    })
+  }
+  fetchSingleMovie = id => {
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
```

Comment:
Could a dynamic fetch function be made? What parameters would you need for this to work for any fetch?